build/pipeline.yml

```diff
@@ -0,0 +1,48 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+pr: none
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: microsoft/vscode-engineering
+      ref: main
+      endpoint: Monaco
+
+parameters:
+  - name: publishPackage
+    displayName: 🚀 Publish vscode-html-languageservice
+    type: boolean
+    default: false
+
+extends:
+  template: azure-pipelines/npm-package/pipeline.yml@templates
+  parameters:
+    npmPackages:
+      - name: vscode-html-languageservice
+
+        buildPlatforms:
+          - name: Linux
+            nodeVersions:
+              - 16.x
+          - name: MacOS
+            nodeVersions:
+              - 16.x
+          - name: Windows
+            nodeVersions:
+              - 16.x              
+
+        buildSteps:
```

Comment:
We need to do `yarn compile` here otherwise it will not be executed before creating the package.

Reply:
`npm test` and `npm run prepublishOnly` both run `npm run compile`.